FlashCap.Core/CaptureDevice.cs

```diff
@@ -66,6 +66,11 @@ public async Task DisposeAsync()
 
     protected abstract Task OnStartAsync(CancellationToken ct);
     protected abstract Task OnStopAsync(CancellationToken ct);
+    public abstract void DisplayPropertyPage_CaptureFilter(IntPtr hwndOwner);
+    public abstract int GetPropertyValue(VideoProcessingAmplifierProperty property);
+    public abstract void SetPropertyValue(VideoProcessingAmplifierProperty property, object? value);
```

Comment:
eventually `SetPropertyValue` if more properties are supported, it should receive a wider `Type/Enum`